00-Starter-Seed/readme.md

```diff
@@ -1,15 +1,22 @@
 # Laravel example tutorial
```

Comment:
What I find confusing is that the QS Article only lists "Login" which would map to "01-Login" in the repo of any other sample, but in this one is still pointing to the "00-Started-Seed" which back then was used to show "how to clone and setup the project", not "how to use it".

I'd suggest:
- Rename this folder to "01-Login" 
- Rename any mention of it on the readme to "Login"
- Each subfolder (each sample) would have a minimal "how to setup" guide, but the actual process, detailed description, etc is on the QS article. So link to it in each Readme. Check this repo's readme and it's subfolders' READMEs to follow what I mean https://github.com/auth0-samples/auth0-android-sample

Reply:
I totally agree, the structure was bothering me as well. I have a PR in progress for the QS so I can change the link there. Is there any other place what I should check for that link?

Reply:
@lbalmaceda - I'll do the folder PR in a different PR

Reply:
It's hard to know. Initially on the quickstarts article, but maybe a blog post is pointing to the wrong link too. @jeffreylees any clue?